pkg/printers/internalversion/printers.go

```diff
@@ -827,11 +827,20 @@ func getServiceExternalIP(svc *api.Service, wide bool) string {
 		lbIps := loadBalancerStatusStringer(svc.Status.LoadBalancer, wide)
 		if len(svc.Spec.ExternalIPs) > 0 {
 			results := []string{}
+			seenResults := make(map[string]bool)
 			if len(lbIps) > 0 {
 				results = append(results, strings.Split(lbIps, ",")...)
 			}
 			results = append(results, svc.Spec.ExternalIPs...)
-			return strings.Join(results, ",")
+			// Avoid repeated addresses
```

Comment:
quite odd that we do this in the printer and not where the object is created. This feels like the printer is lying about the actual object.

Reply:
I agree it's odd.  But we need to have the IP in both places to make it work because in the bare-metal LB we are "cheating" and using the ExternalIP mechanism to implement the LoadBalancer type.  So the IP really should be in two places, but just looks a little silly when it is printed in a comma separated list.